lib/open_project/translations/helpers/tmp_directory.rb

```diff
@@ -0,0 +1,14 @@
+module TmpDirectory
+  def create_tmp_directory(delete: nil)
+    tmp_path = File.join(FileUtils.pwd, 'tmp')
+    if delete
+      FileUtils.rm_rf tmp_path
```

Comment:
As you created a method for it, this could probably also read `remove_tmp_directory(tmp_path)`